src/files/FilesPage.js

```diff
@@ -12,6 +12,7 @@ import withTour from '../components/tour/withTour.js'
 import InfoBoxes from './info-boxes/InfoBoxes.js'
 import FilePreview from './file-preview/FilePreview.js'
 import FilesList from './files-list/FilesList.js'
+import FileNotFound from './file-not-found/FileNotFound.js'
```

Comment:
can we change this file to `file-not-found.tsx` and remove the folder?

ref: https://github.com/ipfs/ipfs-webui/issues/2349